Description: Drop the feature as the tests are failing with it.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -53,7 +53,6 @@
     "dep:image",
     "dep:bytemuck",
 ]
-only_u8x4 = []
 rayon = [
     "dep:rayon",
     "image/rayon",
